src/messages/interfaces/message.interface.ts

```diff
@@ -32,9 +33,17 @@ export interface MessageApp {
 
 export interface MessagePause extends MessageRequiredFields {
   depositRoot: string;
-  nonce: number;
   blockNumber: number;
   blockHash: string;
```

Comment:
Seems we need only blockNumber (no depositRoot and blockHash)
Will new council daemon work correctly with old contracts?

Reply:
I see, here can be a problem, will fix, thanks

Reply:
In the previous version, the message was:

```ts
export interface MessagePause extends MessageRequiredFields {
  depositRoot: string;
  nonce: number;
  blockNumber: number;
  blockHash: string;
  signature: Signature;
  stakingModuleId: number;
}
```
However, only stakingModuleId and blockNumber were signed and used in the pause method. How were the other fields utilized in the Pause bot?

Reply:
If they are not signed, they wasn't used